src/parameters/update_cost_parameters.jl

```diff
@@ -131,7 +132,7 @@ function update_variable_cost!(
     end
     mult_ = parameter_multiplier[component_name, time_period]
     variable = get_variable(container, get_variable_type(attributes)(), T)
-    gen_cost = variable[component_name, time_period] * mult_ * cost_data * base_power * dt
+    gen_cost = variable[component_name, time_period] * (mult_ * cost_data * base_power * dt)
```

Comment:
this is interesting:
the parenthesis forces the floats the be multiplied before and simplified into a single float. the previous one requires a bunch of operations in an expression. not an allocation issue, but slower than multiplying floats.